src/main/java/org/jenkinsci/plugins/ownership/model/branches/OwnershipBranchProperty.java

```diff
@@ -0,0 +1,73 @@
+package org.jenkinsci.plugins.ownership.model.branches;
+
+import com.synopsys.arc.jenkins.plugins.ownership.OwnershipDescription;
+import com.synopsys.arc.jenkins.plugins.ownership.jobs.JobOwnerHelper;
+import hudson.Extension;
+import hudson.model.*;
+import hudson.util.FormValidation;
+import jenkins.branch.*;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.util.Collections;
+
+public class OwnershipBranchProperty extends BranchProperty {
+    
+    private BranchOwnershipStrategy strategy;
+    private String fallbackOwner;
+    
+    @DataBoundConstructor
+    public OwnershipBranchProperty(@Nonnull String fallbackOwner, @Nonnull BranchOwnershipStrategy strategy) {
+        this.strategy = strategy;
+        this.fallbackOwner = fallbackOwner;
+    }
+    
+    public BranchOwnershipStrategy getStrategy() {
+        return strategy;
+    }
+    
+    public String getFallbackOwner() {
+        return fallbackOwner;
+    }
+    
+    @Override
+    @SuppressWarnings("unchecked")
+    public <P extends Job<P, B>, B extends Run<P, B>> JobDecorator<P, B> jobDecorator(final Class<P> clazz) {
+        return new JobDecorator<P, B>() {
+            @Nonnull
+            public P project(@Nonnull P project) {
+                if (project.getParent() instanceof MultiBranchProject && TopLevelItem.class.isAssignableFrom(clazz)) {
+                    MultiBranchProject multiBranchProject = (MultiBranchProject) project.getParent();
+                    Branch branch = multiBranchProject.getProjectFactory().getBranch(project);
+                    
+                    String prospectiveOwner = strategy.determineOwner(branch);
+                    String owner = prospectiveOwner != null ? prospectiveOwner : fallbackOwner;

```

Comment:
inherit the owner from multibranch job?
maybe we could have the fallback owner an optional setting. if unset, we fallback to inheritance of ownership according to global config.